server/methods/groupsList.js

```diff
@@ -0,0 +1,37 @@
+Meteor.methods({
+	groupsList: function(nameFilter, limit, sort) {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'groupsList' });
+		}
+
+		let options = {
+			fields: { name: 1 },
+			sort: { msgs: -1 }
+		};
+
+		//Verify the limit param is a number
+		if (_.isNumber(limit)) {
+			options.limit = limit;
+		}
+
```

Comment:
If we do a limit by default, then paging/skipping needs to also be enabled. I just copied this from the `channelsList` method.